File: src/test/resources/IsStringUnique.java
public class Solution {
    /**
     * @param str input string to be checked
     * @return  Indicate if string contains only unique chars
     */
    public boolean isUniqueChars(String str) {
        if (str == null) return true;

        char[] chars = str.toCharArray();

        for (int i = 0; i < chars.length - 1; i++) {

            char toCheck = chars[i];

            for (int j = i + 1; j < chars.length; j++) {
                if (toCheck == chars[j]) return false;
            }
        }

        return true;
    }
}

File: src/test/resources/IsStringUnique.kt
class IsStringUnique {
    /**
     * @param str input string to be checked
     * *
     * @return  Indicate if string contains only unique chars
     */
    fun isUniqueChars(str: String?): Boolean {
        if (str == null) return true

        val chars = str.toCharArray()

        for (i in 0..chars.size - 1 - 1) {

            val toCheck = chars[i]

            (i + 1..chars.size - 1)
                    .filter { toCheck == chars[it] }
                    .forEach { return false }
        }

        return true
    }
}